resources/views/appointments/index.blade.php

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
```

Comment:
Create layout and restructure the code. Reference: https://laravel.com/docs/8.x/blade